backend/test/Controllers/CodeLookupControllerTest.cs

```diff
@@ -114,6 +115,55 @@ public void GetRoleCodes()
             Model.CodeModel[] resultValue = Assert.IsType<Model.CodeModel[]>(actionResult.Value);
             Assert.Equal(new[] { _mapper.Map<Model.CodeModel>(role) }, resultValue);
         }
+
+        [Fact]
+        public void GetAll()
+        {
+            // Arrange
+            var role = new Entity.Role
+            {
+                Id = Guid.NewGuid(),
+                Name = "Ministry of Health",
+                Description = "The Ministry of Health"
+            };
+            _pimsService.Setup(m => m.Lookup.GetRolesNoTracking()).Returns(new[] { role });
```

Comment:
I'm going to convert all API unit tests in the next sprint to start using better design patterns (see ImportControllerTest).  We need to setup each test properly, right now we're creating one controller/service for all tests.  This can cause undesirable side effects.

Reply:
Agree we need to have a fresh instance on every test